tensorflow/core/kernels/constant_op.cc

```diff
@@ -184,6 +210,43 @@ class FillOp : public OpKernel {
   }
 };
 
+#ifdef TENSORFLOW_USE_SYCL
+template <typename T>
+class FillOp<SYCLDevice, T> : public OpKernel {
+ public:
+  explicit FillOp(OpKernelConstruction* context) : OpKernel(context) {}
+
+  void Compute(OpKernelContext* context) override {
+    const Tensor& Tdims = context->input(0);
+    OP_REQUIRES(
+        context, IsLegacyVector(Tdims.shape()),
+        errors::InvalidArgument("dims must be a vector of int32, got shape ",
+                                Tdims.shape().DebugString()));
+    const Tensor& Tvalue = context->input(1);
+
+    T data_host;
+    auto device = context->eigen_sycl_device();
+    auto size = sizeof(T);
+    auto src_ptr = GetBase(&Tvalue);
```

Comment:
We are aware of the issue and working on resolving it. I removed above workaround in https://github.com/lukeiwanski/tensorflow/commit/1a1525bae9a7e57a3faa2ed8e8a70de9dff3747f where I am registering Fill operation with .HostMemory("value"). Is that acceptable for time being?